test/junit/scala/collection/mutable/ArrayBuilderTest.scala

```diff
@@ -39,4 +39,11 @@ class ArrayBuilderTest {
     (1 to 100).foreach(_ => builder.addAll(Array.empty[String]))
     assertEquals(0, builder.knownSize)
   }
+
+  @Test def `t13068 addAll of array`: Unit = {
+    val ab: ArrayBuilder[Unit] = ArrayBuilder.make[Unit]
+    val arr = Array[Unit]((), (), (), (), (), (), (), (), (), (), (), ())
+    ab.addAll(arr)
+    assertEquals(arr.length, ab.result().length)
```

Comment:
Should we assert the elements  are all ()

Reply:
That sounds extreme, but  `result` doesn't use `Array.fill`. That is an "optimization", but also therefore a potential point of failure. This class "must" be rarely used, so arguably any optimization is not worth it. Look at the time suck due to "optimizing" the range checks. Probably the test should verify the `result`.

Reply:
Thanks for the input, but verify result seems  ✅️

Reply:
but then I would have to decide whether to nowarn
```
[warn] .../ArrayBuilderTest.scala:48:37: dubious usage of method == with unit value
[warn]     assertTrue(ab.result().forall(_ == ()))
[warn]                                     ^
```

Reply:
> but then I would have to decide whether to nowarn
> ```
> [warn] .../ArrayBuilderTest.scala:48:37: dubious usage of method == with unit value
> [warn]     assertTrue(ab.result().forall(_ == ()))
> [warn]                                     ^
> ```

Maybe eq？

Reply:
oh good suggestion. I did add a commit earlier, but didn't push it yet, because I feel bad for Jenkins.